test/test_jit.py

```diff
@@ -5482,6 +5482,37 @@ def fn():
         ast = torch.jit.frontend.get_jit_def(fn)
         self.assertExpected(str(ast))
 
+    def test_python_runtimeexception(self):
+        @torch.jit.script
+        def fn(x):
+            # type: (int) -> int
+            if x > 2:
+                raise RuntimeError("bad input, {} is too high".format(x))
+            return x + 3
+        with self.assertRaisesRegex(torch._C.JITException, "RuntimeError: bad input, 5 is too high"):
```

Comment:
s/torch._C.JITException/torch.jit.Error/

Reply:
Also, how much work would it be to get the class object for the exception type via the PythonResolver at compile time and make it so that when we throw one of these and cross into Python we construct the correct Python exception type?

Reply:
Hmm, well... might not be that hard, but I don't know how to do that 🙃 

I'm not really sure how that would work conceptually, either. Would we need to dynamically import that exception?